docs/serviceWorker.js

```diff
@@ -0,0 +1,90 @@
+---
+---
+
+// Useful resources:
+// - https://eduardoboucas.com/blog/2015/06/04/supercharging-jekyll-with-a-serviceworker.html
+// - https://jakearchibald.com/2014/offline-cookbook
+
+var newCacheName = 'pouchdb-assets-cache-v{{ site.time }}';
+
+var criticalAssets = [
+  '/offline.html'
+  '/static/css/pouchdb.css',
+  '/static/favicon.ico',
+  '/static/js/code.min.js',
+
+  'http://code.jquery.com/jquery.min.js',
+  'http://netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js',
+  'http://cdn.jsdelivr.net/pouchdb/latest/pouchdb.min.js',
+];
```

Comment:
neither is the javascript really, the site fell back really well while these assets didnt load, however the main thing is that after first load are the fonts served from the cache? we definitely want that. It somewhat seems like it should from the `fetch` code below, but I wanted to make sure